### PR TITLE
feat: signer module adds metrics for each rpc interface

### PR DESCRIPTION
**Pull request by sysvm**
### Description

Signer module adds metrics for each rpc interface.

### Rationale

N/A

### Example

N/A

### Changes

Notable changes: 
* N/A

### CODE REVIEW

File: base/gfspapp/sign_server.go [hunk 1]
```diff
@@ -162,6 +162,11 @@ func (g *GfSpBaseApp) GfSpSign(ctx context.Context, req *gfspserver.GfSpSignRequ
 		})
 		if err != nil {
 			log.CtxErrorw(ctx, "failed to create global virtual group", "error", err)
+			metrics.ReqCounter.WithLabelValues(SignerFailureCreateGVG).Inc()
+			metrics.ReqTime.WithLabelValues(SignerFailureCreateGVG).Observe(time.Since(startTime).Seconds())
+		} else {
+			metrics.ReqCounter.WithLabelValues(SignerSuccessCreateGVG).Inc()
+			metrics.ReqTime.WithLabelValues(SignerSuccessCreateGVG).Observe(time.Since(startTime).Seconds())
```
Comment (will-2012) [inline base/gfspapp/sign_server.go:169]: Check other interfaces of the signer and add metrics together.

Comment (sysvm) [reply base/gfspapp/sign_server.go:169]: fixed

File: base/gfspapp/app_options.go [hunk 2]
```diff
@@ -103,32 +103,56 @@ const (
...
-	SignerFailureReceiveTask           = "signer_receive_task_failure"
-	SignerSuccessReplicateApproval     = "signer_secondary_approval_success"
-	SignerFailureReplicateApproval     = "signer_secondary_approval_failure"
-	SignerSuccessRecoveryTask          = "signer_recovery_task_success"
-	SignerFailureRecoveryTask          = "signer_recovery_task_failure"
+	SignerSuccess                           = "signer_success"
+	SignerFailure                           = "signer_failure"
+	SignerSuccessBucketApproval             = "signer_bucket_approval_success"
+	SignerFailureBucketApproval             = "signer_bucket_approval_failure"
+	SignerSuccessMigrateBucketApproval      = "signer_bucket_approval_success"
```
Comment (will-2012) [inline base/gfspapp/app_options.go:110]: migrate bucket.

File: base/gfspapp/app_options.go [hunk 3]
```diff
@@ -103,32 +103,56 @@ const (
...
+	SignerFailureMigrateBucketApproval      = "signer_bucket_approval_failure"
```
Comment (will-2012) [inline base/gfspapp/app_options.go:111]: Ditto.

File: modular/executor/execute_replicate.go [hunk 4]
```diff
@@ -73,6 +71,8 @@ func (e *ExecuteModular) HandleReplicatePieceTask(ctx context.Context, task core
 	sealErr := e.sealObject(ctx, task, sealMsg)
 	metrics.PerfPutObjectTime.WithLabelValues("background_seal_object_cost").Observe(time.Since(sealTime).Seconds())
 	metrics.PerfPutObjectTime.WithLabelValues("background_task_seal_object_end").Observe(time.Since(startReplicateTime).Seconds())
+	metrics.PerfPutObjectTime.WithLabelValues("object_throughout_all_process").Observe(time.Since(
+		time.Unix(task.GetObjectInfo().GetCreateAt(), 0)).Seconds())
```
Comment (will-2012) [inline modular/executor/execute_replicate.go:75]: Also need to add in HandleSealObjectTask.

**Review by will-2012 — APPROVED** [review]

File: base/gfspapp/app_options.go (hunk 2 above)
Comment (sysvm) [reply base/gfspapp/app_options.go:110]: fixed

File: base/gfspapp/app_options.go (hunk 3 above)
Comment (sysvm) [reply base/gfspapp/app_options.go:111]: fixed

File: modular/executor/execute_replicate.go (hunk 4 above)
Comment (sysvm) [reply modular/executor/execute_replicate.go:75]: added

**Pull request merged**